doc-content/enterprise-only/openshift/rhdm711-prod-immutable-kieserver-ref.adoc

```diff
@@ -50,7 +50,7 @@ https://access.redhat.com/documentation/en-us/openshift_container_platform/3.11/
 |`DECISION_CENTRAL_SERVICE` | `WORKBENCH_SERVICE_NAME` | The Service name for the optional Decision Central, where it can be reached, to allow service lookups (for example, maven repo usage), if required. | myapp-rhdmcentr | False
 |`ARTIFACT_DIR` | -- | List of directories from which archives will be copied into the deployment folder. If unspecified, all archives in /target will be copied. | -- | False
 |`KIE_SERVER_MEMORY_LIMIT` | -- | KIE Server Container memory limit. | 2Gi | True
-|`KIE_SERVER_MEMORY_REQUEST` | -- | KIE Server Container memory Request. | 1.5Gi | True
+|`KIE_SERVER_MEMORY_REQUEST` | -- | KIE Server Container memory Request. | 1536Mi | True
```

Comment:
[technical] Most size values are stated in `Gi`, is there a reason for changing this to `1536Mi`?